pages/index.js

```diff
@@ -19,6 +19,8 @@ import BasicCards from "@components/statistics/BasicCards";
 import Button from "@components/Button";
 import Testimonials from "@components/Testimonials";
 import GitHubAccelerator from "@components/GitHubAccelerator";
+import Alert from "@components/Alert";
+import * as alertsData from "@config/app.json";
```

Comment:
As this is the whole config, `alertsData` is misleading

```suggestion
import * as config from "@config/app.json";
```

Reply:
Done!